docs/Legal/Customer Agreement.mdx

```diff
@@ -36,6 +36,22 @@ Si decide utilizar nuestros servicios, acepta pagar las tarifas asociadas con el
 y proporcionarnos un método de pago válido. Si su método de pago falla repetidamente, nos reservamos el derecho de suspender o cancelar su acceso al Servicio, aunque primero se recopilará y enviará una copia de los datos almacenados
 a usted.
 
+### Soporte Técnico
+
+Harmony Ltd. establece una garantía de soporte técnico que comprende la resolución de incidencias, solicitudes de usuarios y RFC (Request for Change) con los siguientes tiempos 
+de respuesta (TTR)*:
```

Comment:
I think it would be more useful to add a table in markdown to be included in the presentation.

Reply:
done